content/management/configuration-api/v3.4/index.mdx

```diff
@@ -4027,6 +4028,114 @@ Informs that a chatting Customer's session fields were updated. The webhook will
 
 </Section>
 
+## Configuration
+
+<Section>
+
+<Text>
+
+### `group_created`
+
+Informs that group has been created on license.
+
+#### Specifics
+
+|                        |                                                                                                                                                                                       |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Action**             | `create_group`                                                                                                                                                               |
+| **Push equivalent in** | [`Agent Chat API`](/messaging/agent-chat-api/v3.4/rtm-reference/#group_created)  |
+
+</Text>
+
+<Code>
+
+<CodeResponse title={'Sample webhook'}>
+
+</CodeResponse>
+
+```json
+{
+  "id": 42,
+  "name": "sales",
+	"language_code": "en",
+	"agent_priorities": {
+    "agent@example.com": "normal",
+    "other_agent@example.com": "first"
+  }
+}
+```
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### `group_updated`
+
+Informs that group has been updated on license.  Only changed fields are sent.
```

Comment:
Same remarks as with `group_created`.
Also, do we use the american double space to separate phrases?